XSLT/nashvillep15769coll19qdctomods.xsl

```diff
@@ -0,0 +1,332 @@
+  <xsl:template match="text()">
+    <xsl:value-of select="normalize-space(.)"/>
+  </xsl:template>
+
+  <!-- match oai_qdc:qualifieddc -->
+  <xsl:template match="oai_qdc:qualifieddc">
+    <!-- document-level variables -->
+    <!-- match the document root and return a MODS record -->
+    <mods xmlns="http://www.loc.gov/mods/v3" version="3.5"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+      <!-- title -->
+      <titleInfo>
+        <xsl:apply-templates select="dc:title"/>
+        <xsl:apply-templates select="dcterms:alternative"/>
+      </titleInfo>
+      <!-- description -->
+      <xsl:apply-templates select="dc:description"/>
+      <!-- creator(s) -->
+      <xsl:apply-templates select="dc:creator"/>
+      <!-- contributor(s) -->
+      <xsl:apply-templates select="dc:contributor"/>
+      <!-- rightsHolder(s) -->
+      <xsl:apply-templates select="dcterms:rightsHolder"/>
+      <!-- subject(s) -->
+      <xsl:apply-templates select="dc:subject"/>
+      <xsl:apply-templates select="dcterms:spatial"/>
+      <!-- language(s) -->
+      <xsl:apply-templates select="dc:language"/>
+      <!-- identifier(s) that are not URLs -->
+      <xsl:apply-templates select="dc:identifier[not(starts-with(normalize-space(.), 'http://'))]"/>
+      <!-- originInfo -->
+      <originInfo>
+        <!-- publisher -->
+        <xsl:apply-templates select="dc:publisher"/>
+        <!-- dcterms:created -->
+        <xsl:apply-templates select="dcterms:created"/>
+        <!-- dcterms:modified -->
+        <xsl:apply-templates select="dcterms:modified"/>
+      </originInfo>
+      <!-- accessCondition -->
+      <accessCondition type="use and reproduction" xlink:href="http://rightsstatements.org/vocab/InC/1.0/">In Copyright</accessCondition>
+      <!-- location: physicalLocation and URLs -->
+      <location>
+        <xsl:apply-templates select="dc:identifier[starts-with(normalize-space(.), 'http://')]"/>
+      </location>
+      <!-- type(s) that start with a capital letter -->
+      <xsl:apply-templates select="dc:type[matches(., '^[A-Z]')]"/>
+      <!-- relatedItem[@type='host'] -->
+      <!-- physicalDescription -->
+      <physicalDescription>
+        <!-- formats -->
+        <xsl:apply-templates select="dc:format"/>
+        <!-- dcterms:extent -->
+        <xsl:apply-templates select="dcterms:extent"/>
+        <!-- type(s) that start with a lower-case letter -->
+        <xsl:apply-templates select="dc:type[matches(., '^[a-z]')]"/>
+      </physicalDescription>
+      <recordInfo>
+        <recordContentSource>Nashville Public Library</recordContentSource>
+        <recordChangeDate><xsl:value-of select="current-date()"/></recordChangeDate>
+        <languageOfCataloging>
+          <languageTerm term="code" authority="iso639-2b">eng</languageTerm>
```

Comment:
`<languageTerm term="code"...` is throwing an error (and I think this is an Old Typo from yours truly); this should be `<languageTerm type="code"...`

Reply:
OOF, yeah, I'll grab this.